engine/graphics/src/test/test_app_vulkan_package_assets.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
```

Comment:
This is what the "embed" feature does for us in the build system.
Do you need something special?
If not, I suggest we use the same pipeline for generating these types of header files.